test/correctness/long_div.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Halide.h"
```

Comment:
We already have https://github.com/halide/Halide/blob/7d836f0f3dbc202a371b8ad55831239c100cb39f/test/correctness/mul_div_mod.cpp

We should retrofit that test to cover this instead of adding a new test.

Reply:
Sure. Will do.